i18n_routable.gemspec

```diff
@@ -18,9 +18,10 @@ Gem::Specification.new do |s|
   s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
-  s.add_dependency "rails", '~> 3.2.0'
-  s.add_dependency 'i18n', '~> 0.6', '< 0.7'
+  s.add_dependency "rails", '~> 4.2.0'
+  s.add_dependency 'i18n', '~> 0.7'
```

Comment:
I think this is going to give us an issue with "en-US" which is an unsupported locale in i18n v0.7